bin/docs.py

```diff
@@ -70,13 +81,24 @@
         if details.get('remarks'):
             docs.write("* %s: _%s_\n" % ('remarks', details['remarks']))
 
+        #link to the license page for each source if page is available
         if details.get('license') and details.get('license').startswith("http"):
             docs.write("* %s: _%s_\n" % ('license', details['license']))
         else:
             docs.write("* %s: `%s`\n" % ('license', details['license']))
 
+        #list out all "via" sources with links to each source's source...
+        if details.get('src:via'):
+            docs.write("\n  This source includes `CC-BY compatible` data from the following organizations:\n")
+            for via in details['src:via']:
+                if via['source_note']:
+                    docs.write("  \t* **%s**: [%s](%s) - %s\n" % (via["context"],via["source_name"],via["source_link"],via["source_note"]))
```

Comment:
Can you add a test for a link and only add the markdown link decorations if there is a link, please?

Here and one down.